conf/airframes/examples/bebop2_indi.xml

```diff
@@ -22,9 +22,7 @@
     <module name="imu" type="bebop"/>
     <module name="gps" type="ublox"/>
     <module name="stabilization" type="indi_simple"/>
-    <module name="ins" type="ekf2">
-      <define name="INS_EKF2_OPTITRACK" value="true"/>
```

Comment:
In other airframes you moved the define to the target, but here is it removed?

Reply:
Yes, since in this airframe, the gps that is declared in `ublox` and not `optitrack`. So the EKF2 flag shouldn't be defined at all. In other airframes, the gps `opitrack` or `datalink` is used for real flights, and is replaced by `nps` for simulation. The EKF2 flag is not compatible with that, so I moved it in `ap` target.